### DISCUSSION TITLE
Where is the weight of ControlNetHED.pth?

### DISCUSSION THREAD

**Discussion by okotaku** [PixArt-alpha/PixArt-alpha#87] [Q&A] [answered]
https://github.com/PixArt-alpha/PixArt-alpha/blob/master/diffusion/model/hed.py#L102

**Comment by lawrence-cj** [comment, marked as answer]
This one: https://huggingface.co/lllyasviel/Annotators/blob/main/ControlNetHED.pth

**Reply by okotaku** [reply]
Thank you!
